app/subscriber/src/features/my-reports/view/ReportSnapshotEdit.tsx

```diff
@@ -33,6 +33,18 @@ export const ReportSnapshotEdit: React.FC = () => {
     [generateReport, setValues],
   );
 
+  const handleExport = React.useCallback(async () => {
+    try {
+      if (instance?.id) {
+        await toast.promise(exportReport(instance?.id), {
+          pending: 'Downloading file',
```

Comment:
Might be overkill but if we have some sort of status update message for other downloads in the application we could make an enum for this